tools/ramclustr/ramclustr_wrapper.R

```diff
@@ -3,9 +3,9 @@ store_output <- function(ramclustr_obj,
                          output_spec_abundance,
                          msp_file) {
   RAMClustR::write.msp(ramclustr_obj, one.file = output_merge_msp)
-  write.csv(ramclustr_obj$SpecAbund,
+  write.table(ramclustr_obj$SpecAbund,
     file = output_spec_abundance,
-    row.names = TRUE, quote = FALSE
+    row.names = TRUE, quote = FALSE, col.names = NA, sep = "\t"
```

Comment:
Doesn't `col.names = NA` lead to the column names not being written to the file?

Reply:
in write.table function it does not have this meaning as per the [documentation](https://www.rdocumentation.org/packages/utils/versions/3.6.2/topics/write.table)
_"By default there is no column name for a column of row names. If col.names = NA and row.names = TRUE a blank column name is added, which is the convention used for CSV files to be read by spreadsheets."_